Userland/Libraries/LibIPC/Connection.h

```diff
@@ -42,6 +42,7 @@ class ConnectionBase : public Core::Object {
 
     void set_fd_passing_socket(NonnullOwnPtr<Core::Stream::LocalSocket>);
     void set_deferred_invoker(NonnullOwnPtr<DeferredInvoker>);
+    DeferredInvoker& deferred_invoker() { return *m_deferred_invoker; }
```

Comment:
Any reason for this not to be `const` ?

Reply:
I need to call non-const functions on the deferred invoker